tests/throttle-test.ts

```diff
@@ -123,6 +123,23 @@ QUnit.test('throttled function is called with final argument', function(assert)
   bb.throttle(null, throttled, 'bus' , 10, false);
 });
 
+QUnit.test('throttle returns same timer', function(assert) {
```

Comment:
Can you confirm that this test failed prior to the fix above?

Reply:
Yes 😜